AUTHORS

```diff
@@ -122,7 +123,9 @@ Michi Mutsuzaki                         michi@isovalent.com
 Moh Ahmed                               moh.ahmed@cengn.ca
 Moritz Johner                           beller.moritz@googlemail.com
 Moshe Immerman                          moshe.immerman@vitalitygroup.com
+Nathan Bird                             njbird@infiniteenergy.com
 Nathan Taylor                           ntaylor1781@gmail.com
+Neela Jacques                           68304471+Neelajacques@users.noreply.github.com
```

Comment:
I couldn't figure out how to get this to show up with the right address, so I just left this as-is.

Reply:
Scratch that, I just had the emails around the wrong way in .mailmap. Fixed.